conf/application.conf

```diff
@@ -86,6 +86,7 @@ microservice {
       company-auth-frontend {
         external-url = "http://localhost:9025"
         sign-in.path = "/gg/sign-in"
+        sign-in.continue-url="/agent-mapping/start"
```

Comment:
Does this need the `http://localhost:9438/` preamble to support local dev like the other continue URL below?

Reply:
yeah, i have picked up on this, it was sitting in the frontendAppConfig instead